Yosemite/Yosemite/Tools/Media/MediaAssetExporter.swift

```diff
@@ -123,7 +124,8 @@ private extension MediaAssetExporter {
     /// - parameter onError: Called if an error was encountered during export.
     ///
     private func exportGIF(forAsset asset: PHAsset, resource: PHAssetResource, onCompletion: @escaping MediaExportCompletion) {
-        guard UTTypeEqual(resource.uniformTypeIdentifier as CFString, kUTTypeGIF) else {
+        let gifTypeIdentifier = UTType.gif.identifier as CFString
+        guard UTTypeEqual(resource.uniformTypeIdentifier as CFString, gifTypeIdentifier) else {
```

Comment:
nit: similar to above, I think direct string comparison is fine?

```suggestion
        guard resource.uniformTypeIdentifier == UTType.gif.identifier else {
```